pkg/detector/detector.go

```diff
@@ -75,7 +75,9 @@ type ResourceDetector struct {
 	// waitingObjects tracks of objects which haven't be propagated yet as lack of appropriate policies.
 	waitingObjects map[keys.ClusterWideKey]struct{}
 	// waitingLock is the lock for waitingObjects operation.
-	waitingLock sync.RWMutex
+	waitingLock                         sync.RWMutex
+	ConcurrentReconciles                int
```

Comment:
```suggestion

	// ConcurrentResourceTemplateSyncs is the number of resource templates that are allowed to sync concurrently.
	// Larger number means responsive resource template syncing but more CPU(and network) load.
	ConcurrentResourceTemplateSyncs     int
```
This flag is just for the resource template, for the RB/PP we have other flags, right?

Reply:
yes

Reply:
Seems no update.